node/src/actors/chain_manager/handlers.rs

```diff
@@ -780,7 +786,7 @@ impl Handler<PeersBeacons> for ChainManager {
                         self.initialize_from_storage(ctx);
                         log::info!("Restored chain state from storage");
 
-                        self.sm_state = StateMachine::WaitingConsensus;
+                        self.sm_state = StateMachine::AlmostSynced;
```

Comment:
If your peers has a consensus and you are out of them, you have to restore from storage and move to WaitingConsensus not AlmostSynced

Reply:
veeery true